contrib/guix/README.md

```diff
@@ -0,0 +1,229 @@
+# Bootstrappable Bitcoin Core Builds
```

Comment:
drop 15277 for now, maybe open in new PR, it will require significantly more review and changes